project/Dependencies.scala

```diff
@@ -10,7 +10,7 @@ object Dependencies {
     val RuntimeVersion = System.getProperty(
       "kalix-runtime.version",
       // temporarily accept the old system property name
-      System.getProperty("kalix-proxy.version", "1.1.46"))
+      System.getProperty("kalix-proxy.version", "1.2.0"))
```

Comment:
If we are bumping here, should we also bump the remaining places? Or maybe we do on a separate PR?

Reply:
Ah, I didn't notice that from Ciscos original PR, hmm. Might be fine to do here, to make sure tests are run against the new version. We anyway want that bump.